_posts/2021-04-09-prace-workshop.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: MDAnalysis Workshop @ PRACE / SURF
+---
+
+MDAnalysis will be delivering a 3 half-day remote workshop hosted by PRACE / SURF on the 26-28th of May.
+
+Workshop topics will range from the fundamentals of MDAnalysis all the way through to how to create your own AnalysisBase-derived methods and parallel analysis in a HPC setting.
+
+
+## Register soon [places limited]
+
+The workshop is open to everyone, but places are limited, apply early!
```

Comment:
Our target audience is users in academia and industry who are already familiar with Python and molecular dynamics. 

^ Can we shove that in anywhere?